### PR TITLE
TouchEvent support

### PR DESCRIPTION
**Pull request by jacereda**

### CODE REVIEW

**Comment by jacereda** [comment]
Should I leave the `DOM.HTML.Event.EventTypes` usage out of this PR maybe?

File: src/Halogen/HTML/Events.purs [hunk 1]
```diff
@@ -41,31 +41,34 @@ module Halogen.HTML.Events
...
-import DOM.Event.Event as EE
-import DOM.HTML.Event.Types (DragEvent)
-
-import Halogen.HTML.Core as Core
 import Halogen.HTML.Core (Prop)
 import Halogen.HTML.Properties (IProp)
 import Halogen.Query (Action, action)
 import Halogen.Query.InputF (InputF(..))
-
 import Unsafe.Coerce (unsafeCoerce)
```
Comment (garyb) [inline src/Halogen/HTML/Events.purs]: Could you restore the alphabetical sorting of imports please?

**Comment by jacereda** [comment]
No idea about the build failure, works fine locally.

**Comment by garyb** [comment]
Thanks!

**Pull request merged**